src/test/java/com/reminiscence/reminiscence/userAccount/testUserAccountSetters.java

```diff
@@ -0,0 +1,100 @@
+package com.reminiscence.reminiscence.userAccount;
+
+import com.reminiscence.reminiscence.account.AccountRepo;
+import com.reminiscence.reminiscence.account.UserAccount;
+import com.reminiscence.reminiscence.journal.Entry;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class testUserAccountSetters {
+
+    private UserAccount user1;

```

Comment:
I feel like the `@Before` was overused in these tests.  Keeping your code "DRY" is important, but a lot of times it's better to create everything a test needs in isolation.

Also:  `@After` could have been used to cleanup the repo and removing the entries.  Right now, many of these tests will fail if ran "all at once" because the db doesn't get cleaned up.